backend/packages/identity-scan/src/scan/events/index.js

```diff
@@ -11,18 +19,9 @@ async function handleEvents(events = [], blockIndexer, extrinsics = []) {
       eventIndex,
     };
 
-    const { event, phase } = events[eventIndex];
-    let extrinsic;
-    if (!phase.isNone) {
-      const extrinsicIndex = phase.value.toNumber();
-      indexer = {
-        ...indexer,
-        extrinsicIndex,
-      };
-      extrinsic = extrinsics[extrinsicIndex];
-    }
```

Comment:
I suggest keep the extrinsic, and pass it to events handling methods, because we may need some info from the extrinsic.